src/i18n/locales/en.json

```diff
@@ -18,12 +18,12 @@
     "title": "Secure, verifiable and flexible digital voting"
   },
   "banner_voting_types": {
-    "anonymous_description": "Say goodbye to the high expenses of traditional voting methods. Vocdoni App brings you a more affordable way to conduct elections, cutting down on logistical and administrative costs while maintaining the highest standards of quality and security.",
+    "anonymous_description": "Say goodbye to the high expenses of traditional voting methods. {{app}} brings you a more affordable way to conduct elections, cutting down on logistical and administrative costs while maintaining the highest standards of quality and security.",
     "anonymous_title": "Cost-Effective Voting Solutions",
     "bottom_text": "Privacy should be normal",
-    "flexible_description": "Engagement is key in democratic processes, and our platform is designed to maximize it. With its user-friendly interface and accessibility, Vocdoni App encourages higher participation rates, ensuring a more representative and inclusive voting experience.",
+    "flexible_description": "Engagement is key in democratic processes, and our platform is designed to maximize it. With its user-friendly interface and accessibility, {{app}} encourages higher participation rates, ensuring a more representative and inclusive voting experience.",
     "flexible_title": "Boosted Participation Rates",
-    "token_description": "At the heart of Vocdoni App is our commitment to security. Leveraging cutting-edge cryptographic methods, we ensure that every vote cast is not only secure but fully verifiable. This transparency builds unparalleled trust among participants, crucial for the legitimacy of any election.",
+    "token_description": "At the heart of {{app}} is our commitment to security. Leveraging cutting-edge cryptographic methods, we ensure that every vote cast is not only secure but fully verifiable. This transparency builds unparalleled trust among participants, crucial for the legitimacy of any election.",
```

Comment:
I cannot find these strings in the vocdoni app. Are we sure we're using it everywhere that needs a variable for the app name? 

Note I've no problem with this, since I personally had to do it before, but I'm just wondering if it was really required here.

Reply:
This strings are used on the vocdoni app landing. 
![image](https://github.com/vocdoni/ui-scaffold/assets/16777076/9abc62f6-54e6-42c6-90fb-74f616c0e6df)

Is for that that i added the app variable.